### PR TITLE
Ccd 2661 step 1

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-2661

### Change description ###
Generalise names of commonly used configs

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by kaanaktas — APPROVED** [review]
lgtm

**Review by danlysiak — APPROVED** [review]

File: src/contractTest/resources/application.properties [hunk 1]
```diff
@@ -31,17 +31,17 @@ pagination.page.size=25
...
 definition.cache.jurisdiction-ttl=${DEFINITION_CACHE_JURISDICTION_TTL_SEC:30}
 
 #max size of cache map in terms of number of entries
-definition.cache.max.size=${DEFINITION_CACHE_MAX_SIZE:5000}
+default.cache.max.size=${DEFAULT_CACHE_MAX_SIZE:5000}
 
-#eviction policy. If set to NONE the cache can grow arbitrarily large and definition.cache.max.size is ignored
-definition.cache.eviction.policy=${DEFINITION_CACHE_EVICTION_POLICY:NONE}
+#eviction policy. If set to NONE the cache can grow arbitrarily large and max size is ignored
+default.cache.eviction.policy=${DEFAULT_CACHE_EVICTION_POLICY:NONE}
```
Comment (mattnayler) [inline src/contractTest/resources/application.properties:44]: `definition.cache.jurisdiction-ttl` and `definition.cache.max-idle.secs` not updated as per `src/main/resources/application.properties`.    If they need to match may be cut and past the section so order and comments match as well.

**Review by mattnayler — APPROVED** [review]
Approved: with minor comment on `src/contractTest/resources/application.properties`

File: src/contractTest/resources/application.properties (hunk 1 above)
Comment (mattnayler) [reply src/contractTest/resources/application.properties:44]: Sorry: `definition.cache.jurisdiction-ttl` doesn't need to change ...

... but `definition.cache.max-idle.secs` probably should, i.e. to `default.cache.max-idle`.

Comment (MSancaktutar) [reply src/contractTest/resources/application.properties]: Done. Thank you, Matt.

**Comment by MSancaktutar** [comment]
Merged through https://github.com/hmcts/ccd-data-store-api/pull/1928 already - hence closing this one.

**Pull request closed without merging**